packages/react-storybook-addon/preset.js

```diff
@@ -1,17 +1,9 @@
 function config(entry = []) {
-  // This entrypoint is deliberately set to ts for monorepo DX
-  // However this is bad for publishing
-  // https://github.com/microsoft/fluentui/issues/20474
-  // https://github.com/microsoft/fluentui/issues/18357
-  return [...entry, require.resolve('./src/preset/preview.ts')];
+  return [...entry, require.resolve('./lib/preset/preview')];
```

Comment:
making this work is just not possible because there is no auto transpilation happening when storybook loads presets